folium/folium.py

```diff
@@ -88,6 +88,10 @@ class Map(LegacyMap):
         Forces Leaflet to not use hardware-accelerated CSS 3D
         transforms for positioning (which may cause glitches in some
         rare environments) even if they're supported.
+    custom_css : string array, default []
```

Comment:
I wonder if overloading the `Map` class is the best way to do it.
Also, why do you need an `[]` as default? I'd rather use `None`.